css/how_to_name_the_classes.md

```diff
@@ -0,0 +1,118 @@
+| --------- | ---------- | ----------- |
+| niño      | nino       | kid         |
+| schön     | schon      | beautiful   |
+| délicieux | delicieux  | delicious   |
+
+### B. Prefix conflicts
+
+To prevent conflicts with other projects, define a prefix with the name of the module you are working on, and use it for every class that belong to that context. If the name is too long, use an abbreviation of it. If the prefix is too common, combine it with the project's name.
+
+```scss
+// SCSS example working on Fjord theme
+.fjord-button {...}
+.fjord-image {...}
+.fjord-text {...}
+```
+
+### C. Suffix modifiers
+
+Keep the name as simple as possible and add suffixes to create modifiers of the class
+
+```scss
+// Class base SCSS
+.fjord-btn {...} // stands for: .fjord-button
+```
+
+The general rule is `context-component-modifier`
+
+```scss
+// Class modifiers SCSS
+.fjord-btn-sm {...} // stands for: .fjord-button-small
+.fjord-btn-lg {...} // stands for: .fjord-button-large
+```
+
+If the variations of the class are too many, use a numeric suffix
+
+```scss
+// Class numeric modifiers SCSS
+.fjord-text-1 {...}
+.fjord-text-2 {...}
+.fjord-text-3 {...}
+.fjord-text-4 {...}
+```
+
+### D. Component instances
+
+We always recommend the solution proposed in the previous section, but sometimes it will be necessary to change the standard method
+
+> Imagine we need to create 2 images in 2 different components for the Fjord theme, but the differences between these elements are so uniques that we can't afford to generate the required modifiers because they won't be used anywhere else.
+
+In that case we are going to rely on the Atomic Design concepts to add the desired specifications to the class.
+
+```scss
+// Class instance SCSS
+.fjord-card-image {...}
+.fjord-list-image {...}
+```
+
+### E. Responsive particles
+
+There is a special part of the name is worth mentioning even if we created a dedicated session for this topic: the "responsive particles". [future_link]
+
+The "responsive particle" is a part of the class that can be `sm`, `md`, `lg`, or `xl` (small, medium, large, or extra-large), and it goes between the `class-property-word` and the `class-value-word`
+
+```scss
+// SCSS example
+.col-sm-6 // create a column that occupy 6/12 of the available container width on devices with a screen width of `576px` or above
+.col-md-4 // create a column that occupy 4/12 of the available container width on devices with a screen width of 768px or above
+.mt-3 // add a top margin of 16px on every device
+.mt-md-4 // add a top margin of 24px on devices with a screen width of 768px or above
+.d-lg-none // hide the element on devices with a screen width of 992px or above
+.d-xl-block // show the element on devices with a screen width of 1200px or above
+```
+
+We should create our responsive classes following the same pattern
+
+```scss
+// SCSS example
+.fjord-title-sm-1 {
+	font-size: 20px;
+}
+.fjord-title-md-2 {
+	font-size: 30px;
+}
```

Comment:
Should these examples be, in fact, responsive? I was expecting to see some media-queries in here to see how `-md-` and `-sm-` would affect different devices. Without it, this seems more or less arbitrary.